app/services/app-update.js

```diff
@@ -0,0 +1,35 @@
+// @flow
```

Comment:
This service/util only serves the `AppUpdater` component ATM right? If that's the case, IMO it's better to have it in the same directory.

Thoughts?

Reply:
My reasoning was to  collocate functionality that make side-effects such as api calls closer together (in the services dir). But you are right this can easily live in the AppUpdate component directory.